pkg/event/target/queuestore.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	maxLimit = 10000 // Max store limit.
```

Comment:
using the same variable for `maxLimit` looks confusing @Praveenrajmani we should say perhaps defaultLimit?